terraform/infrastructure/azure/main.tf

```diff
@@ -179,11 +181,14 @@ module "loadbalancer_backend_worker" {
   ports                          = []
 }
 
+# We cannot delete them right away since we first need to to delete the dependency from the VMSS to this backend pool.
+# TODO(@3u13r): Remove this dependency after the next Constellation release.
```

Comment:
Could you please rephrase the TODOs such that they're discoverable with, say, `git grep -E "TODO.*v2\.[0-9]+"`?